src/prototypes/article-listing/example/example.twig

```diff
@@ -0,0 +1,145 @@
+                    <p<a href="#">{{article.author}}</a></p>
+                    <p>{{article.pubDate|date('M j, Y')}}</p>
+                  </div>
+                </div>
+              {% endblock %}
+            {% endembed %}
+          {% endfor %}
+        {% endblock %}
+      {% endembed %}
+    </div>
+  </div>
+  {% for topic in featured.topics %}
+    <div class="o-container o-container--pad">
+      <div class="o-container__content">
+        {% embed '@cloudfour/objects/deck/deck.twig' with {
+          class: 'o-deck--3-column@l',
+        } %}
+          {% block content %}
+            <div class="title-content">
+              <h2 class="section-title">{{topic.title}}</h2>
+              <div>
+                {% embed '@cloudfour/components/button/button.twig' with { class: 'c-button--secondary' } %}
+                  {% block content %}{{topic.cta}}{% endblock %}
+                  {% block content_end %}
+                    {% include '@cloudfour/components/icon/icon.twig' with {
+                      "name": "arrow-right"
+                    } only %}
+                  {% endblock %}
+                {% endembed %}
+              </div>
+            </div>
+            {% for article in topic.articles %}
+              {% embed '@cloudfour/components/card/card.twig' with {
+                class: '',
+                href: article.link
+              } %}
+                {% block heading %}
+                  {{article.title}}
+                {% endblock %}
+                {% block cover %}
+                  <img src="{{article.thumbnail}}" alt="">
+                {% endblock %}
+                {% block content %}
+                  <p>
+                    {% if article.description|length > 140 %}
+                      {{article.description|slice(0, 140)|trim}}…
+                    {% else %}
+                      {{article.description}}
+                    {% endif %}
+                  </p>
+                {% endblock %}
+                {% block footer %}
+                  <div class="c-author o-media">
+                    <div class="o-media__object">
+                      {% include '@cloudfour/components/avatar/avatar.twig' with { src: article.avatar } only %}
+                    </div>
+                    <div class="o-media__content">
+                      <p<a href="#">{{article.author}}</a></p>
+                      <p>{{article.pubDate|date('M j, Y')}}</p>
+                    </div>
+                  </div>
+                {% endblock %}
+              {% endembed %}
+            {% endfor %}
+          {% endblock %}
+        {% endembed %}
+      </div>
+    </div>
+  {% endfor %}
+  <div class="o-container o-container--pad">
+    <div class="o-container__content">
+      <div class="topics">
+        <h2 class="section-title">More Topics</h2>
+        <div class="search-content">
+          <label for="search-input" class="search-label">Search</label>
```

Comment:
Something is feeling unresolved about this currently.

Should the button and form field be joined? Can the icon be sized a little more nicely? Is the label necessary, and in the right place? 🤔